grafana-init/grafana.py

```diff
@@ -121,31 +122,32 @@ def create_dashboard_payload(json_path):
 
 
 def main():
-    logging.info('Opening a Grafana session...')
-    session = Session()
-    login(session)
-
-    if check_initialized(session):
-        logging.info('Grafana has already been initialized, skipping!')
-        return
-
-    logging.info('Attempting to add configured datasource...')
-    r = session.post('{url}/api/datasources'.format(url=GRAFANA_URL),
-                     json=create_datasource_payload())
-    logging.debug('Response: %r', r.json())
-    r.raise_for_status()
-
-    for path in sorted(glob.glob('{dir}/*.json'.format(dir=DASHBOARDS_DIR))):
-        logging.info('Creating dashboard from file: {path}'.format(path=path))
-        r = session.post('{url}/api/dashboards/db'.format(url=GRAFANA_URL),
-                         json=create_dashboard_payload(path))
+    for user in range(len(create_login_payload())):
```

Comment:
I think it's better to change here like below.
```
for user in create_login_payload():
```

Need to change L.74 as below.
```
json=user,
```